index.html

```diff
@@ -6,6 +6,7 @@
   </head>
   <body>
     <main id='main'></main>
+    <link rel="stylesheet" href="node_modules/react-table/react-table.css">
```

Comment:
yes. or you can use `insert-css` again with [`style-resolve`](https://github.com/stackcss/style-resolve)